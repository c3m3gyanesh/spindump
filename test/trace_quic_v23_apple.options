--textual --report-spins --report-notes --format text