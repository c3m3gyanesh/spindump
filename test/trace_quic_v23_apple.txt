
A test between two Apple QUIC endpoints, provided by rpaulo on October 16, 2019. This is two normal connections. Spin bit is spinning.
