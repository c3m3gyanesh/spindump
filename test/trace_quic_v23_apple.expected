QUIC ::1 <-> ::1 08e5adb4f4bc0b18-9beb43407a17e16f (49940:4433) at 1571246326860201 new starting packets 0 0 bytes 0 0 note "No rsp,v23,no spin"
QUIC ::1 <-> ::1 08e5adb4f4bc0b18-08a1315d498ef248 (49940:4433) at 1571246326874356 measurement up right 14155 packets 1 0 bytes 1264 0 bandwidth 1264 0 note "No rsp,v23,no spin"
QUIC ::1 <-> ::1 e5adb4f4bc0b1815-08a1315d498ef248 (49940:4433) at 1571246326881454 spinvalue up 0 initiator packets 7 6 bytes 3037 3101 bandwidth 3037 3101 note "V23,no spin"
QUIC ::1 <-> ::1 e5adb4f4bc0b1815-08a1315d498ef248 (49940:4433) at 1571246326881635 spinvalue up 0 initiator packets 8 6 bytes 3339 3101 bandwidth 3339 3101 note "V23,no spin"
QUIC ::1 <-> ::1 e5adb4f4bc0b1815-08a1315d498ef248 (49940:4433) at 1571246326881808 spinvalue up 0 responder packets 9 6 bytes 3423 3101 bandwidth 3423 3101 note "V23,no spin"
QUIC ::1 <-> ::1 e5adb4f4bc0b1815-08a1315d498ef248 (49940:4433) at 1571246326882578 spinvalue up 0 responder packets 9 7 bytes 3423 3408 bandwidth 3423 3408 note "V23,no spin"
QUIC ::1 <-> ::1 e5adb4f4bc0b1815-08a1315d498ef248 (49940:4433) at 1571246326882679 spinvalue up 1 initiator packets 9 8 bytes 3423 3487 bandwidth 3423 3487 note "V23,no spin"
QUIC ::1 <-> ::1 e5adb4f4bc0b1815-08a1315d498ef248 (49940:4433) at 1571246326883292 spinvalue up 1 responder packets 10 8 bytes 3511 3487 bandwidth 3511 3487 note "V23,no R-spin"
QUIC ::1 <-> ::1 e5adb4f4bc0b1815-08a1315d498ef248 (49940:4433) at 1571246326883292 measurement up right 613 packets 10 8 bytes 3511 3487 bandwidth 3511 3487 note "V23,spinning"
QUIC ::1 <-> ::1 e5adb4f4bc0b1815-08a1315d498ef248 (49940:4433) at 1571246326883447 spinvalue up 0 initiator packets 10 9 bytes 3511 3618 bandwidth 3511 3618 note "V23,spinning"
QUIC ::1 <-> ::1 e5adb4f4bc0b1815-08a1315d498ef248 (49940:4433) at 1571246326883447 measurement up left 155 packets 10 9 bytes 3511 3618 bandwidth 3511 3618 note "V23,spinning"
QUIC ::1 <-> ::1 e5adb4f4bc0b1815-08a1315d498ef248 (49940:4433) at 1571246326883447 measurement up full (initiator) 768 packets 10 9 bytes 3511 3618 bandwidth 3511 3618 note "V23,spinning"
QUIC ::1 <-> ::1 e5adb4f4bc0b1815-08a1315d498ef248 (49940:4433) at 1571246326883760 spinvalue up 0 responder packets 11 9 bytes 3590 3618 bandwidth 3590 3618 note "V23,spinning"
QUIC ::1 <-> ::1 e5adb4f4bc0b1815-08a1315d498ef248 (49940:4433) at 1571246326883760 measurement up right 313 packets 11 9 bytes 3590 3618 bandwidth 3590 3618 note "V23,spinning"
QUIC ::1 <-> ::1 e5adb4f4bc0b1815-08a1315d498ef248 (49940:4433) at 1571246326883760 measurement up full (responder) 468 packets 11 9 bytes 3590 3618 bandwidth 3590 3618 note "V23,spinning"
QUIC ::1 <-> ::1 0871f474b3486599-df601e50a6049a73 (49941:4433) at 1571246326884881 new starting packets 0 0 bytes 0 0 note "No rsp,v23,no spin"
QUIC ::1 <-> ::1 0871f474b3486599-0820161a6107d772 (49941:4433) at 1571246326898878 measurement up right 13997 packets 1 0 bytes 1264 0 bandwidth 1264 0 note "No rsp,v23,no spin"
QUIC ::1 <-> ::1 e5adb4f4bc0b1815-08a1315d498ef248 (49940:4433) at 1571246326899536 spinvalue up 0 responder packets 11 10 bytes 3590 3695 bandwidth 3590 3695 note "V23,spinning"
QUIC ::1 <-> ::1 71f474b34865995c-0820161a6107d772 (49941:4433) at 1571246326900367 spinvalue up 0 initiator packets 6 5 bytes 2942 2970 bandwidth 2942 2970 note "V23,no spin"
QUIC ::1 <-> ::1 71f474b34865995c-0820161a6107d772 (49941:4433) at 1571246326900422 spinvalue up 0 initiator packets 7 5 bytes 3240 2970 bandwidth 3240 2970 note "V23,no spin"
QUIC ::1 <-> ::1 e5adb4f4bc0b1815-08a1315d498ef248 (49940:4433) at 1571246326900531 spinvalue up 1 initiator packets 11 11 bytes 3590 3772 bandwidth 3590 3772 note "V23,spinning"
QUIC ::1 <-> ::1 e5adb4f4bc0b1815-08a1315d498ef248 (49940:4433) at 1571246326900531 measurement up left 16771 packets 11 11 bytes 3590 3772 bandwidth 3590 3772 note "V23,spinning"
QUIC ::1 <-> ::1 e5adb4f4bc0b1815-08a1315d498ef248 (49940:4433) at 1571246326900531 measurement up full (initiator) 17084 packets 11 11 bytes 3590 3772 bandwidth 3590 3772 note "V23,spinning"
QUIC ::1 <-> ::1 71f474b34865995c-0820161a6107d772 (49941:4433) at 1571246326900649 spinvalue up 0 responder packets 8 5 bytes 3327 2970 bandwidth 3327 2970 note "V23,no spin"
QUIC ::1 <-> ::1 71f474b34865995c-0820161a6107d772 (49941:4433) at 1571246326901207 spinvalue up 0 responder packets 8 6 bytes 3327 3277 bandwidth 3327 3277 note "V23,no spin"
QUIC ::1 <-> ::1 71f474b34865995c-0820161a6107d772 (49941:4433) at 1571246326901278 spinvalue up 1 initiator packets 8 7 bytes 3327 3356 bandwidth 3327 3356 note "V23,no spin"
QUIC ::1 <-> ::1 71f474b34865995c-0820161a6107d772 (49941:4433) at 1571246326901849 spinvalue up 1 responder packets 9 7 bytes 3415 3356 bandwidth 3415 3356 note "V23,no R-spin"
QUIC ::1 <-> ::1 71f474b34865995c-0820161a6107d772 (49941:4433) at 1571246326901849 measurement up right 571 packets 9 7 bytes 3415 3356 bandwidth 3415 3356 note "V23,spinning"
QUIC ::1 <-> ::1 71f474b34865995c-0820161a6107d772 (49941:4433) at 1571246326901928 spinvalue up 0 initiator packets 9 8 bytes 3415 3470 bandwidth 3415 3470 note "V23,spinning"
QUIC ::1 <-> ::1 71f474b34865995c-0820161a6107d772 (49941:4433) at 1571246326901928 measurement up left 79 packets 9 8 bytes 3415 3470 bandwidth 3415 3470 note "V23,spinning"
QUIC ::1 <-> ::1 71f474b34865995c-0820161a6107d772 (49941:4433) at 1571246326901928 measurement up full (initiator) 650 packets 9 8 bytes 3415 3470 bandwidth 3415 3470 note "V23,spinning"
QUIC ::1 <-> ::1 e5adb4f4bc0b1815-08a1315d498ef248 (49940:4433) at 1571246336892598 spinvalue up 1 initiator packets 12 11 bytes 3669 3772 bandwidth 3669 3772 note "V23,spinning"
QUIC ::1 <-> ::1 71f474b34865995c-0820161a6107d772 (49941:4433) at 1571246336907127 spinvalue up 0 initiator packets 10 8 bytes 3494 3470 bandwidth 3494 3470 note "V23,spinning"
